test/unit/clock_lowres.c

```diff
@@ -0,0 +1,37 @@
+#include "unit.h"
+
+#include <math.h>
+#include "clock_lowres.h"
+#include "clock.h"
+
+#if defined(NDEBUG) && !defined(__APPLE__)
+#define CLOCK_LOWRES_RESOLUTION 0.05
+#else
+#define CLOCK_LOWRES_RESOLUTION 0.1
+#endif
```

Comment:
Why do you use a different resolution for debug builds? Anyway, IMO better export the resolution from the module rather than redefining it in tests and keeping those values in sync.

Reply:
agree, done